distribution/lib/Standard/Base/0.0.0-dev/src/System/File_Format.enso

```diff
@@ -220,7 +220,10 @@ type Plain_Text_Format
        Implements the `File.read` for this `File_Format`
     read : File -> Problem_Behavior -> Any
     read self file on_problems:Problem_Behavior =
-        file.read_text (self.resolve_encoding file) on_problems
+        result = file.read_text (self.resolve_encoding file) on_problems
+        result.if_not_error <|
+            Telemetry.log "File_Format.read" "Read file: format={}, output={}" ["Plain_Text", "Text"]
+        result
```

Comment:
Btw while not for this, looking at the `if_not_error`, I guess actually logging the errors could also be quite valuable (perhaps ignoring expected ones like File Not Found)

Reply:
agree - that is probably a nice addition to the `File_Format.read` that we log when we fail.